### PR TITLE
Add two deathmatch zones for Lag Shot fights

### PR DESCRIPTION
**Pull request by RussellLVP**
Thank you for making a contribution to [Las Venturas Playground](https://sa-mp.nl/)!

Please take a moment to fill in the questions in this template, to make it easier for our developers
to understand what you're doing.

## What is being changed?
Branch the two most popular `/deathmatch` areas, Fight Club and the Baseball field, in skin hit and lag shot variants. The new zones have IDs `6` and `7`.

## Why is this being changed?
To enable players to have fights with their preferred lag compensation mode.

## How are you making the change?
  [X] I have a working check-out of Las Venturas Playground.
  [X] I have included tests for any changed JavaScript code.
  [X] I have tested this change myself on my local server.

### CODE REVIEW

**Comment by OttoRocket-LVP** [comment]
Great work :) Yay for having both lagshot and lagcomp deathmatches! 👍

File: data/death_match_locations.json [hunk 1]
```diff
@@ -26,6 +26,34 @@
         ],
         "boundaries": [ 2110.8208, 2055.1360, 2200.4045, 2144.6980 ]
     },
+    {
+        "id": 6,
```
Comment (OttoRocket-LVP) [inline data/death_match_locations.json]: I think we'll want to keep this ordered on ID?

File: data/death_match_locations.json [hunk 2]
```diff
@@ -26,6 +26,34 @@
...
+        "name": "LV RW FightClub (lag shot)",
```
Comment (OttoRocket-LVP) [inline data/death_match_locations.json]: LV fightclub isn't really a popular fight zone for RW. (Due to it being very small, so only works for a small amount of players)
Maybe the slot is better used @ SF Military? As that one is quite often used.

File: data/death_match_locations.json (hunk 1 above)
Comment (RussellLVP) [reply data/death_match_locations.json]: Sure, I grouped them by area. Updated.

File: data/death_match_locations.json (hunk 2 above)
Comment (RussellLVP) [reply data/death_match_locations.json]: Good call. Updated.

**Review by OttoRocket-LVP — APPROVED** [review]

**Pull request merged**